### ISSUE TITLE
base_temperature

### ISSUE THREAD

**Issue by kibok90** [HobbitLong/SupContrast#1]
Thanks for sharing the code!

What is the intuition behind `base_temperature`? I found that its value is always `0.07`. Is it just for scaling the learning rate by `temperature / 0.07`?
https://github.com/HobbitLong/SupContrast/blob/master/losses.py#L95

**Comment by HobbitLong** [comment]
It just scales up the loss, you can ignore it.

**Comment by Ywandung-Lyou** [comment]
Should it be a sensitive hyper-parameter that should be fine-tuned carefully on the dataset?

**Comment by FraCorti** [comment]
> Should it be a sensitive hyper-parameter that should be fine-tuned carefully on the dataset?

This is something that I'm also struggling to understand. Did you understand if it should be fine-tuned or not?

**Comment by KaiChen1998** [comment]
> > Should it be a sensitive hyper-parameter that should be fine-tuned carefully on the dataset?
> 
> This is something that I'm also struggling to understand. Did you understand if it should be fine-tuned or not?

By dividing self.temperature first, and then times (self.temperature / self.base_temperature), it seems like self.temperature is useless and the actual temperature is always self.base_temperature, which is always 0.07, different from the default temperature 0.1 according to the README file?
